sklearn/tree/tests/test_export.py

```diff
@@ -94,12 +113,12 @@ def test_graphviz_toy():
                 'fontname=helvetica] ;\n' \
                 'edge [fontname=helvetica] ;\n' \
                 '0 [label=<X<SUB>0</SUB> &le; 0.0<br/>samples = 100.0%<br/>' \
-                'value = [0.5, 0.5]>, fillcolor="#e5813900"] ;\n' \
-                '1 [label=<samples = 50.0%<br/>value = [1.0, 0.0]>, ' \
+                'proportion = [0.5, 0.5]>, fillcolor="#e5813900"] ;\n' \
```

Comment:
In case of using proportion = 'both' there are two separate fields - value and proportion. Because of that I changed value to proportion for case proportion=True, to have consistency here.